api/e2e/test/cluster/cluster.go

```diff
@@ -41,7 +41,7 @@ var clients = &struct {
 type TuringRouterResources struct {
 	KnativeServices []knservingv1.Service
 	K8sServices     []coreV1.Service
-	IstioServices   []v1alpha3.VirtualService
+	IstioServices   []*v1alpha3.VirtualService
```

Comment:
This is now changed to a list of pointers because the new Istio Go client returns a list of pointers instead of `VirtualService` objects - see https://github.com/caraml-dev/turing/blob/9823c2d2fda90e1c21dbc2c8bf829145e0c97498/api/e2e/test/cluster/cluster.go#L154.